terraform/modules/tf_stream_alert/main.tf

```diff
@@ -180,3 +180,21 @@ resource "aws_cloudwatch_log_metric_filter" "alert_processor_cw_metric_filters"
     value     = "${element(split(",", var.alert_processor_metric_filters[count.index]), 2)}"
   }
 }
+
+// CloudWatch metric alarms that are created per-cluster
+// The split list is our way around poor tf support for lists of maps and is made up of:
+// <alarm_name>, <alarm_description>, <comparison_operator>, <evaluation_periods>,
+// <metric>, <period>, <statistic>, <threshold>
```

Comment:
This seems like a pretty good workaround. Maybe add a TODO so we can come back and update this once Terraform has better support for complex types